src/luarocks/cmd.lua

```diff
@@ -328,7 +328,7 @@ Variables:
 ]]..get_config_text()
 
    local parser = argparse(
-      program_name, "LuaRocks "..cfg.program_version..", the Lua package manager\n\n"..
+      dir.base_name(program), "LuaRocks "..cfg.program_version..", the Lua package manager\n\n"..
```

Comment:
Is this correct for `luarocks-admin`? (isn't `program` only correct for luarocks itself?)

Reply:
It works for luarocks-admin since that's a separate binary.